week1/Program.cs

```diff
@@ -0,0 +1,89 @@
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+app.MapGet("/", () => "This is the homework for week1!");
+
+//task1
+app.MapGet("/string-manipulation", (string input) => {
+string reversed = ReverseString(input); 
```

Comment:
The formatting is a bit off in the file, you can use VSCode formatter from C# plugin to fix this (key combination is ALT+SHIFT+F on windows).